### DISCUSSION TITLE
Random seed

### DISCUSSION THREAD

**Discussion by Astrosofica** [lllyasviel/Omost#60] [Ideas]
Hi lllyasviel,

I really love your script, it's awesome.
I was a bit frustrated by the random seed not being random. So I adjusted the gradio_app.py file.

Just after the original imports I added:

```
import random

# Function to generate a random seed
def generate_random_seed():
    return random.randint(0, 2**31 - 1)
```

A second addition I added just after line 312 (in my edited file), just behind the line: 

            seed = gr.Number(label="Random Seed", value=12345, precision=0)

I added:

```
 random_seed_btn = gr.Button("Generate Random Seed", variant="secondary", size="sm", min_width=60)
            
 # Update the seed value when the button is clicked
 random_seed_btn.click(fn=generate_random_seed, outputs=seed)
```

This will create a "Generate Random Seed" button that you can click when needed.

Just an idea, I'm curious what you think about it.
Best regards, Jan.

**Comment by dfl** [comment]
@Astrosofica implemented your suggestion in PR #85
